builtin/mainmenu/dlg_clients_list.lua

```diff
@@ -0,0 +1,48 @@
+-- Luanti
+-- Copyright (C) 2024 siliconsniffer
+-- SPDX-License-Identifier: LGPL-2.1-or-later
+
+
+local function clients_list_formspec(dialogdata)
+	local clients_list = dialogdata.server.clients_list
+	local servername   = dialogdata.server.name
+
+	local function fmt_formspec_list(clients_list)
+		local escaped = {}
+		for i, str in ipairs(clients_list) do
+			escaped[i] = core.formspec_escape(str)
+		end
+		return table.concat(escaped, ",")
+	end
+
+	local formspec = {
+		"formspec_version[8]",
+		"size[6,9.5]",
+		"position[0.5,0.5]",
```

Comment:
this is the default, no need to repeat it

Reply:
also, to make the formspec a bit nicer on mobile, I suggest adding

```lua
local TOUCH_GUI = core.settings:get_bool("touch_gui")
```

and

```suggestion
		TOUCH_GUI and "padding[0.01,0.01]" or "",
```